### PR TITLE
added python_requires to setup.py (fixes #77)

### LINKED ISSUE #77: Add a Python 3.5 version floor
Right now, the fact that this project only supports Python 3.5 and up is only documented in [trove classifiers](https://pypi.org/pypi?%3Aaction=list_classifiers). This provides documentation but no enforcement.

Should add a version floor explicitly in `setup.py`.

### PR DESCRIPTION
**Pull request by lorenafreitas**
This should fix #77 . 

Let me know if you'd prefer it done some other way!

### CODE REVIEW

**Comment by lorenafreitas** [comment]
Oops, that's embarrassing – you got me, I'm a python newbie. Thanks for being patient, I'll fix it straight away :-)

**Comment by jameslamb** [comment]
> Oops, that's embarrassing – you got me, I'm a python newbie. Thanks for being patient, I'll fix it straight away :-)

No problem! Trust me, `setup.py` is real confusing (especially if you're used to R's package management stuff).

File: setup.py [hunk 1]
```diff
@@ -32,6 +32,7 @@
     maintainer='James Lamb',
     maintainer_email='jaylamb20@gmail.com',
     install_requires=runtime_deps,
+    python_requires='>=3.5',
     extras_require={
```
Comment (jameslamb) [inline setup.py]: could you please remove the space before the `,`?

Comment (jameslamb) [reply setup.py]: oh wait maybe it just looks like a space on mobile haha nevermind!

**Pull request merged**